rwengine/src/audio/SfxParameters.hpp

```diff
@@ -0,0 +1,133 @@
+#ifndef _RWENGINE_SFX_PARAMETERS_HPP_
+#define _RWENGINE_SFX_PARAMETERS_HPP_
+
+/// Nr of existing sfx sounds
+constexpr size_t kNrOfAllSfx = 3032;
```

Comment:
What is the source of this number? Isn't this data dependent?

Reply:
Taken from gtamodding.